integration/setup/common.go

```diff
@@ -44,48 +49,149 @@ func common(ctx context.Context, config Config) error {
 	}
 
 	{
-		c := chartvalues.CertOperatorConfig{
-			CommonDomain:       env.CommonDomain(),
-			RegistryPullSecret: env.RegistryPullSecret(),
-			Vault: chartvalues.CertOperatorVault{
-				Token: env.VaultToken(),
-			},
-		}
-
-		values, err := chartvalues.NewCertOperator(c)
+		err := installCertOperator(ctx, config)
```

Comment:
Install `cert-operator` from control plane catalog, instead of using e2etemplates.

Reply:
Why shouldn't we install from control plane catalog using e2etemplates? It would be good to share those values in case other project needs to install it too.

Reply:
We had a chat outside GitHub. The reason is that it's hard to do all the vendoring dance to propagate a single value update.